### DISCUSSION TITLE
More conditional check in Condition Block

### DISCUSSION THREAD

**Discussion by Bullkn0x** [postmanlabs/postman-flows#216] [Ideas]
Would love to be able to make a check against the length of a List as a conditional or whether a given object contains a key

**Comment by saswatds** [comment]
Hey @Bullkn0x, we have added the `size` and `len` operators. https://github.com/postmanlabs/postman-flows/releases/tag/v22.01.2 
In the next few released I will continue adding more operators

**Reply by Bullkn0x** [reply]
IS 9.9.4 live yet?  In app update check says 9.3 is latest and I can't see anywhere on the web where I can upgrade

**Reply by Bullkn0x** [reply]
Found it in the web app.  Any idea when this is this going live on Mac desktop?

**Reply by saswatds** [reply]
The desktop app rollout happens a little bit slowly by an automated system because of how network-intensive the process is. Hopefully, you should get it in a day or two. The platform teams are working on a solution to consolidate the web and desktop delivery patterns, hoping that happens soon.
